tests/e2e/e2e_workflow.spec.js

```diff
@@ -174,8 +174,37 @@ describe('Basic end-to-end Workflow', function () {
       expect(result.exitCode).to.be.equal(0);
 
       // Clean `tmp` folder
-      process.chdir(path.resolve(root, '..'));
-      sh.rm('-rf', './tmp'); // tmp.teardown takes too long
+      // process.chdir(path.resolve(root, '..'));
+      // sh.rm('-rf', './tmp'); // tmp.teardown takes too long
+    });
+  });
+  
+  it('Turn on `noImplicitAny` in tsconfig.json and rebuild', function (done) {
+    this.timeout(420000);
+    
+    var configFilePath = path.join(process.cwd(), 'src', 'tsconfig.json');
+    fs.readFile(configFilePath, 'utf8', function(err, data){
+      
+      var config = JSON.parse(data);
+      config.compilerOptions.noImplicitAny = true;
+      
+      fs.writeFile(configFilePath, JSON.stringify(config), function(){
+        //clear the dist folder
+        sh.rm('-rf', path.join(process.cwd(), 'dist'));
+        
+        return ng([
```

Comment:
No, not needed. It was a result of copying in the promise implementation from above. But at the same time, right now that does not hurt anything. The next change I make in there, I'll clean that up, but I don't think it's worth a separate PR to remove it with no other changes.